InventoryUpdator.java

```diff
@@ -66,14 +251,20 @@ public void validateStock(int quantity){
                 stockOK = true;
             }
         }
+
+        return stockOK;
     }
 
     //This will create an interface with the user to upload their file
     public void massOrdering(){}
 
     //This will create an interface with the user to make a single order
     public void individualOrdering(){
+    public void individualOrdering() throws FileNotFoundException {

```

Comment:
There is two "public void individualOrdering(){", was one meant to be removed?

Reply:
Good catch! Just removed it